src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -113,10 +113,7 @@
  * Default value is {@code "^messages.*$"}.
  * </li>
  * <li>
- * Property {@code fileExtensions} - Specify file type extension to identify
- * translation files. Setting this property is typically only required if your
- * translation files are preprocessed and the original files do not have
- * the extension {@code .properties}
+ * Property {@code fileExtensions} - Specify the file extensions of the files to process.
```

Comment:
Even we loose a bit of details, I still think it is ok. I do not think this is critical details.

Reply:
I do not agree here. Why do we need to change this?

Reply:
Because content comes from abstract class method, it is same for many other Checks.

Reply:
```java
 * <p>
 * Note: If your project uses preprocessed translation files and the original files do not have the
 * {@code properties} extension, you can specify additional file extensions via the {@code fileExtensions} property.
 * </p>


```

Reply:
Please add this to check description then ^

Reply:
done